rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1092,6 +1092,8 @@ def test_upgrade_db_35_to_36(user_data_dir):  # pylint: disable=unused-argument
         ('0x2B888954421b424C5D3D9Ce9bB67c9bD47537d12', 'hardware'),
         ('1PUrJgftNnHvvqVyEsm9DiCDQuZHCn47fQ', 'hot'),
     ]
+    cursor.execute('SELECT * FROM address_book')
+    assert cursor.fetchall() == [('0xc37b40ABdB939635068d3c5f13E7faF686F03B65', 'ETH', 'yabir secret account')]  # noqa: E501
```

Comment:
Can you maybe both here and in global add 1 more? Just to see that we don't have off by one errors and that everything is transferred.

Whenever I add data to the test DB for upgrade I always add more than 1 exactly for this reason.